pkg/planner/core/operator/logicalop/logical_apply.go

```diff
@@ -136,9 +139,7 @@ func (la *LogicalApply) DeriveStats(childStats []*property.StatsInfo, selfSchema
 		RowCount: leftProfile.RowCount,
 		ColNDVs:  make(map[int64]float64, selfSchema.Len()),
 	})
-	for id, c := range leftProfile.ColNDVs {
```

Comment:
I have do this linter at https://github.com/pingcap/tidb/pull/60406. I find this changement will create bug. so we can revert it.

Reply:
I have reverted it and added a comment.